jfrog_platform_cloud/assets/logs/jfrog_artifactory.yaml

```diff
@@ -27,96 +22,160 @@ facets:
-    path: usr.id
+  - facetType: range
+    groups:
+      - Measure
+    name: Duration
+    path: duration
     source: log
+    type: double
+    unit:
+      family: time
+      name: nanosecond
   - groups:
-      - JFrog Cloud
-    name: Tenant ID
-    path: jfrog.cloud.tenant_id
+      - Event
+    name: Event Name
+    path: evt.name
     source: log
-  - groups:
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Data Changed
-    path: jfrog.cloud.data_changed
+    name: Access repository path
+    path: repository_path
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Request Duration
-    path: jfrog.cloud.duration
+    name: Action Response
+    path: jfrog.cloud.action_response
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Request Content Length
-    path: jfrog.cloud.request_content_length
+    name: Audit Data Changed
+    path: jfrog.cloud.audit_data_changed
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Logged Principal
-    path: jfrog.cloud.logged_principal
+    name: Company Name
+    path: jfrog.cloud.company_name
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Token ID
-    path: jfrog.cloud.token_id
+    name: Entity Name
+    path: jfrog.cloud.entity_name
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Repository Name
-    path: jfrog.cloud.repo
+    name: Event Type
+    path: jfrog.cloud.evt.type
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
     name: Image Name
     path: jfrog.cloud.image
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Entity Name
-    path: jfrog.cloud.entity_name
+    name: Instance Name
+    path: jfrog.cloud.instance_name
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
       - JFrog Cloud
-    name: Event Name
-    path: jfrog.cloud.evt.name
+    name: Logged Principal
+    path: jfrog.cloud.logged_principal
     source: log
-  - groups:
+    type: string
+  - description: Artifactory package type
+    facetType: list
+    groups:
       - JFrog Cloud
-    name: Response Content Length
-    path: jfrog.cloud.response_content_length
+    name: Package Type
+    path: jfrog.cloud.package_type
     source: log
-  - groups:
+    type: double
+  - description: Artifactory Project Name
+    facetType: list
+    groups:
       - JFrog Cloud
-    name: Event Type
-    path: jfrog.cloud.evt.type
+    name: Project Name
+    path: jfrog.cloud.project_key
     source: log
-  - groups:
+    type: string
+  - facetType: list
+    groups:
+      - JFrog Cloud
+    name: Repository Name
+    path: jfrog.cloud.repo
```

Comment:
I don't see any remapper to this path neither tests in the sample, are you sure its being used ?

Reply:
There is a remapper for that attribute. It's [here](https://github.com/DataDog/integrations-extras/blob/0cef2b0b7189fc56253c9012ab1a7fad211e9fef/jfrog_platform_cloud/assets/logs/jfrog_artifactory.yaml#L181). 
We have `jfrog.cloud.repo` and `jfrog.cloud.image` attributes for docker-related logs, the sample was not for the docker pull or push event. 
I'm fixing it in the next commit.